Adapters/AdapterBehaviorStandardization/StandardizedActionsAndTriggers.md

```diff
@@ -1,547 +1,371 @@
 # Descriptions of standardized actions or triggers
-**Version Publish Date: 01.03.2018**
+**Version Publish Date: 05.12.2018**
 
-**Semantic Version of Document: 1.0.0**
+**Semantic Version of Document: 2.0.0**
 
 # Table of Contents
-- [Standardized Triggers (including webhooks)](#standardized-triggers-including-webhooks)
-  * [Bulk Extract](#bulk-extract)
-  * [Get Objects Polling](#get-objects-polling)
-  * [Get Objects Webhook](#get-objects-webhook)
-  * [Get Deleted Objects Polling](#get-deleted-objects-polling)
-  * [Get Deleted Objects Webhook](#get-deleted-objects-webhook)
```

Comment:
Why remove the webhook descriptions?

Reply:
It's still there, just in the non-standardized section.  The reason for that is that webhook is that the general webhook case is receive data and pass it along.

Reply:
I saw that it is listed there but we had a description for what it is and what the differences with regards to polling triggers are. Couldn't find this information in the new version.

Reply:
I added that information here: https://github.com/openintegrationhub/Connectors/pull/40